miasm2/ir/ssa.py

```diff
@@ -322,3 +322,20 @@ def reassemble_expr(self, e):
                 if id_rhs in self.expressions:
                     todo.add(id_rhs)
         return e
+
+
+class SSAPath(SSABlock):
```

Comment:
This class never seems to be used. May a `map` of `SSABlock.transform` is enough, and then this class useless for now